Update GEOS-Chem version to 14.6.0 release

msulprizio opened the pull request:
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard

### Describe the update

The GEOS-Chem version is updated to the latest release (14.5.0) in `run_imi.sh`. 

The script `run_boundary_conditions.sh` has also been updated to utilize the latest release version of GCClassic instead of hardcoding to a specific release.

nicholasbalasus commented:
Hi Melissa – we'll need to create a new set of boundary conditions before merging this because of changes to the methane simulation. Do we have space constraints for hosting multiple versions on AWS @laestrada?

I'd like to switch to a strategy of scaling the GEOS-Chem vertical profiles based on ∆(GC-TROPOMI) as opposed to removing this bias uniformly from each of the vertical layers of GC which could be good to pair with this update.

msulprizio commented:
Thanks @nicholasbalasus. In that case, would it be better to wait for the 14.5.1 release which will contain a fix for [livestock seasonality](https://github.com/geoschem/geos-chem/issues/2513)?

nicholasbalasus commented:
I think that makes sense!

msulprizio commented:
OK. I have changed the title of this PR and removed @nicholasbalasus and @laestrada as reviewers. I'll request reviews again when we're ready to update to 14.5.1 following its release.

msulprizio commented:
I am closing this PR. See #335 instead which updates the IMI to use 14.6.1.

nicholasbalasus commented:
Thanks, Melissa! Could you remove the change from the BCs script for now?
I'll address that in a separate PR (presumably for IMI 2.2.1).

On Wed, Jun 4, 2025 at 2:41 PM Melissa Sulprizio ***@***.***>
wrote:

> *msulprizio* left a comment (geoschem/integrated_methane_inversion#293)
> <https://github.com/geoschem/integrated_methane_inversion/pull/293#issuecomment-2941041422>
>
> I am closing this PR. See #335
> <https://github.com/geoschem/integrated_methane_inversion/pull/335>
> instead which updates the IMI to use 14.6.1.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/geoschem/integrated_methane_inversion/pull/293#issuecomment-2941041422>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV5ZFYYRMCSJPNLT7ANR76D3B44T7AVCNFSM6AAAAABTC24OM2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBRGA2DCNBSGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

msulprizio commented:
> Thanks, Melissa! Could you remove the change from the BCs script for now? I'll address that in a separate PR (presumably for IMI 2.2.1).

Yep, that script has been restored in commit d9a2b804f9270c74c90b5f29a60eae73f3289b97. Sorry about that!

nicholasbalasus commented:
Thank you!

On Wed, Jun 4, 2025 at 2:48 PM Melissa Sulprizio ***@***.***>
wrote:

> *msulprizio* left a comment (geoschem/integrated_methane_inversion#293)
> <https://github.com/geoschem/integrated_methane_inversion/pull/293#issuecomment-2941062594>
>
> Thanks, Melissa! Could you remove the change from the BCs script for now?
> I'll address that in a separate PR (presumably for IMI 2.2.1).
>
> Yep, that script has been restored in commit d9a2b80
> <https://github.com/geoschem/integrated_methane_inversion/commit/d9a2b804f9270c74c90b5f29a60eae73f3289b97>.
> Sorry about that!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/geoschem/integrated_methane_inversion/pull/293#issuecomment-2941062594>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV5ZFY2ORY3S37ET2RMZ7733B45OBAVCNFSM6AAAAABTC24OM2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNBRGA3DENJZGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
